src/main/java/org/folio/marc4ld/service/ld2marc/field/Bibframe2MarcFieldRule.java

```diff
@@ -0,0 +1,24 @@
+package org.folio.marc4ld.service.ld2marc.field;
+
+import java.util.Collection;
+import org.folio.ld.dictionary.PredicateDictionary;
+import org.folio.ld.dictionary.model.Resource;
+import org.folio.marc4ld.service.ld2marc.field.param.ControlFieldParameter;
+import org.folio.marc4ld.service.ld2marc.field.param.SubFieldParameter;
+
+public interface Bibframe2MarcFieldRule {
+
+  String getTag();
+
+  boolean isSuitable(Resource resource, PredicateDictionary predicate);
+
+  boolean isSatisfied(Resource resource);
```

Comment:
This method returns a boolean value to indicate if DataField can be created out of Resouce. So, I think `canCreateDataField` or `isDataFieldCreatable` may be a better name.